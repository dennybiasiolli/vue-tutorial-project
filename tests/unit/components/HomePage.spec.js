import { shallowMount } from '@vue/test-utils'
import { expect, it } from 'vitest'
import HomePage from '@/components/HomePage.vue'

it('renders as expected', () => {
  const wrapper = shallowMount(HomePage)
  expect(wrapper.html()).toMatchSnapshot()
})
