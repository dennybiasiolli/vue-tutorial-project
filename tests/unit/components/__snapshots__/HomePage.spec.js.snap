// Vitest Snapshot v1

exports[`renders as expected 1`] = `"<h1>Home Page</h1>"`;
